### ISSUE TITLE
Classification inconsistencies in Analytics category

### ISSUE THREAD

**Issue by andydavies** [patrickhulce/third-party-web#108]
@patrickhulce I've been looking at a adding the other RUM products as per https://discuss.httparchive.org/t/who-are-the-top-rum-analytics-providers/15/25 but notices inconsistencies in the way some of the existing products are classified.

Some RUM scripts (SpeedCurve, mPulse) are classified as `analytics`, others (AppDynamics, New Relic and RayGun) are classified `utilities` and they all really belong in the same category.

As a separate thought I also wonder if the analytics category is too wide – GA, Yandex Metrica etc. are very different to FullStory, HotJar, Decibel Insight etc. which are very different to RUM products (at least in their current implementations) – but can see issues with creating extra categories, has here been any thoughts on sub-categories?

**Comment by patrickhulce** [comment]
Yes we definitely dont want more top-level categories. I can see an argument for some of these being included in both analytica and utilities.

We would love subcategories (see discussion in https://github.com/patrickhulce/third-party-web/issues/55) but its a large effort to recategorize every entity and would require an upfront taxonomy.

Do you have a proposal for a new master set of subcategories?

**Comment by andydavies** [comment]
Happy to propose a new master set of subcategories but it'll take me a while as I'm planning on taking August off ATM

I think I agree with @rockeynebhwani in #55 that experimentation is probably a separate category to analytics, though there are blurry lines.
